<launch>

    <node pkg="aa241x_vision" type="vision_node" name="vision_node" output="screen">
        <param name="frame_width" value="1280" />
        <param name="frame_height" value="720" />
        <param name="display_video" value="true" />
    </node>

    <!-- 
    <node pkg="aa241x_vision" type="display_node" name="display_node" output="screen">
        <param name="frame_width" value="1280" />
        <param name="frame_height" value="720" />
    </node>
    -->
    



</launch>
